examples/bri-3/src/bri/zeroKnowledgeProof/zeroKnowledgeProof.module.ts

```diff
@@ -5,6 +5,7 @@ import { CCSMAnchorHashStorageAgent } from './agents/ccsmAnchorHashStorage.agent
 import { CCSMAnchorHashController } from './api/ccsmAnchorHash.controller';
 import { CreateCCSMAnchorHashCommandHandler } from './capabilities/createCCSMAnchorHash/createCCSMAnchorHashCommand.handler';
 import { VerifyCCSMAnchorHashCommandHandler } from './capabilities/verifyCCSMAnchorHash/verifyCCSMAnchorHashCommand.handler';
+import { BlockchainService } from './services/blockchain/blockchain.service';
```

Comment:
@ognjenkurtic can we be consistent with naming, please `Blockchain` --> `CCSM`

Reply:
agree with both rename and also tackling it in separate PR since this PR is not introducing the service

renaming will increase size of PR and even though renaming is basically insta-approve, if we include it here these "real" changes won't be detected easily in bunch of files, so let's do that rename in separate PR

Reply:
true ... let's merge and open issue to tackle the renaming @ognjenkurtic